### PR TITLE
updating home page with updated content

### PR DESCRIPTION
**Pull request by mansona**
This is just splitting @MelSumner 's work from #558 into different PRs 👍

Fixes https://github.com/ember-learn/ember-website/issues/551

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -215,7 +215,7 @@
         class="lg:col-2 text-center"
         @icon="books"
       >
-        <h3>Books</h3>
+        <h4 class="text-md">Books</h4>
```
Comment (MelSumner) [inline app/templates/index.hbs]:
```suggestion
        <h4 class="text-md">Books & blogs</h4>
```

File: app/templates/index.hbs [hunk 2]
```diff
@@ -215,7 +215,7 @@
...
 
         <ul class="list-unstyled">
           <li><a href="https://www.balinterdi.com/rock-and-roll-with-emberjs/">Rock and Roll with Ember.js</a></li>
```
Comment (MelSumner) [inline app/templates/index.hbs]: There's other content missing here

**Comment by mansona** [comment]
So I added the "books and blogs" but the content was supposed to be in a different PR that I forgot to open 🙈 sorry

https://github.com/ember-learn/ember-website/pull/563

We should merge this PR and leave the other one for the discussion with the learning team 👍

File: app/templates/index.hbs [hunk 3]
```diff
@@ -217,7 +217,7 @@
         class="lg:col-2 text-center"
         @icon="books"
       >
-        <h3>Books</h3>
+        <h4 class="text-md">Books & blogs</h4>
 
         <ul class="list-unstyled">
           <li><a href="https://www.balinterdi.com/rock-and-roll-with-emberjs/">Rock and Roll with Ember.js</a></li>
```
Comment (MelSumner) [inline app/templates/index.hbs]:
```suggestion
          <li><a href="https://www.balinterdi.com/rock-and-roll-with-emberjs/">Rock and Roll with Ember.js</a></li>
          <li><a href="http://emberatlas.com/" title="unofficial resource that provides additional information for developers">Ember Atlas</a></li>
          <li><a href="https://www.pzuraq.com/tag/octane/" title="a blog series by pzuraq">Octane: the blog series</a></li>
```

**Comment by mansona** [comment]
I'm sorry if i'm not being clear here 🙈 I have a **separate** PR for the content of the books and blogs section over here https://github.com/ember-learn/ember-website/pull/563 because the content of that section is a bit contentious right now and needs to be cleared by the learning team to be added.

Does this make sense? 

I would like to see this PR merged and not have it blocked on the content of the contentious section (another reason why I am in favour of smaller PRs)

**Comment by MelSumner** [comment]
I think we got it sorted out, I don't think it has to be a separate PR. 👍

**Pull request merged**